templates/usr/local/sbin/aptly_mirror_sync.sh.erb

```diff
@@ -19,3 +19,9 @@ if [[ $? -ne 0 ]]; then
 else
   /usr/bin/aptly publish switch -passphrase="<%= @gpg_pass -%>" -batch=true -component=non-free,main,contrib ${mirror_name} ${mirror_name}-nonfree-${date} ${mirror_name}-main-${date} ${mirror_name}-contrib-${date}
 fi
+
+# remove old snapshots related to this mirror
+for snapshot in $( /usr/bin/aptly snapshot list -raw=true | grep -v ${date} | egrep "^${mirror_name}-" ); do /usr/bin/aptly snapshot drop $snapshot; done
```

Comment:
after some discussion in slack it appears that the drop snapshot command will work - it will just mean it doesn't immediately clean all snapshots - on the next run it will drop any snapshots that were merged from